.gitignore

```diff
@@ -22,3 +22,4 @@ bin/
 
 .DS_Store
 .localeapp
+.tools-version
```

Comment:
@bitboxer I think it would be good to have a `.tool-versions` in the repo. If you want to add that in a separate PR, I'd be happy to merge it. We could probably add versions for Postgres and some other stuff, though Ruby is fine for now.

Side note: asdf-ruby has [a setting for it to also use `.ruby-version`](https://github.com/asdf-vm/asdf-ruby#ruby-version-file).

Reply:
Ah, thanks. TIL 👍. I removed the `.tools-version` from this commit. Will create a new PR with it and all necessary tools when I am back from my vacation.